spec/features/datepicker_spec.rb

```diff
@@ -3,6 +3,15 @@
 RSpec.feature "visitor selects a date" do
   include_examples "feature helper"
 
+  before :all do
+    next_tuesday = (Time.new + 1.week).beginning_of_week.at_noon + 1.day
```

Comment:
It *is* variable: http://api.rubyonrails.org/classes/DateAndTime/Calculations.html#method-i-beginning_of_week

I'd suggest using `beginning_of_week(:monday)`, though it wouldn't surprise me if this assumption is pervasive.